template/expander.go

```diff
@@ -24,8 +28,21 @@ type Expander struct {
 
 	Resolver   string
 	FailAction string
+
+	fs afero.Fs
+}
+
+func NewExpander(fs afero.Fs) Expander {
+	if fs == nil {
+		fs = genDefaultFs()
```

Comment:
Since `genDefaultFs()` isn't being used anywhere else, this can probably just be `fs = afero.NewOsFs()`.